Source/WebCore/rendering/svg/RenderSVGInlineText.cpp

```diff
@@ -85,7 +85,11 @@ String RenderSVGInlineText::originalText() const
 
 void RenderSVGInlineText::setRenderedText(const String& text)
 {
-    RenderText::setRenderedText(text);
+    auto const textWithWS = applySVGWhitespaceRules(text, style().whiteSpace() == WhiteSpace::Pre);
```

Comment:
`applySVGWhitespaceRules()` returns a String so there's no need to null check on the next line. This can just be:
`RenderText::setRenderedText(applySVGWhitespaceRules(text, style().whiteSpace() == WhiteSpace::Pre))`

Reply:
But `RenderSVGInlineText::styleDidChange()` calls `setText(applySVGWhitespaceRules(originalText(), false), true)` so without further investigation I don't know what the relationship is between `setText` and `setRenderedText`.

Reply:
> `applySVGWhitespaceRules()` returns a String so there's no need to null check on the next line. This can just be:
> `RenderText::setRenderedText(applySVGWhitespaceRules(text, style().whiteSpace() == WhiteSpace::Pre))`

I did exactly that - `RenderText::setRenderedText(applySVGWhitespaceRules(text, style().whiteSpace() == WhiteSpace::Pre)` in previous PR and it passed the test but you mentioned that it might lead to performance issues, hence, I opted for this approach.

Reply:
> But `RenderSVGInlineText::styleDidChange()` calls `setText(applySVGWhitespaceRules(originalText(), false), true)` so without further investigation I don't know what the relationship is between `setText` and `setRenderedText`.

Should I check with @nikolaszimmermann for his input?